assets/src/edit-story/components/library/panes/media/mediaUtils.js

```diff
@@ -95,7 +229,11 @@ export const getResourceFromUploadAPI = (file) => {
   const {
     guid: { rendered: src },
     mime_type: mimeType,
-    media_details: { width, height },
+    media_details: {
+      width: oWidth, // TODO: check why we are using this format `oWidth` on media library and improve it
```

Comment:
What kind of improvement are we expecting here?

Reply:
It's a side note to consider from my code.